tests/svgmin.js

```diff
@@ -127,6 +123,6 @@ test('should emit an error when the Vinyl object is in stream mode', async (t) =
 test('should emit an error when svgo throws an error', async (t) => {
     await t.throwsAsync(makeTest(null, 'file does not contain an SVG'), {
         instanceOf: PluginError,
-        message: /Error in parsing SVG/,
+        message: /SvgoParserError/,
```

Comment:
Why change the error message?

Reply:
So when I updated SVGO they removed this error and converted it to be an SvgoParserError. Here is the commit they did the change: https://github.com/svg/svgo/commit/77102ed0963a5f1597e8072536311b55465e1874

And the exact lines and file lib/svgo/svg2js.js

![Screen Shot 2021-10-04 at 12 51 39](https://user-images.githubusercontent.com/6673572/135907732-a288b62d-e5ea-4de8-8fbb-a12a60c51864.png)

Reply:
If there's a better regex or something I can edit it. But this is the new error message that svgo does